<launch>
  <arg name="gui" default="True" />
  <arg name="in_dir" default="$(find amp_motion_conversion)/cfg/motions/sie"/>
  <arg name="out_dir" default="$(find amp_motion_conversion)/cfg/motions/sie/converted"/>
  <arg name="playback_speed" default="0.35"/>
  <arg name="store_converted" default="False"/>
  
  <param name="use_gui" value="$(arg gui)"/>
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" ></node>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  
  <arg name="rvizconfig" default="$(find amp_motion_conversion)/cfg/rviz.rviz" />
  <param name="robot_description" textfile="$(find amp_motion_conversion)/urdf/humanoid2.urdf"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true"  />

</launch>
